src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1041,37 +1123,11 @@ private bool CheckFieldValueKind(SyntaxNode node, BoundFieldAccess fieldAccess,
                 // S has a mutable field x, then c.f.x is not a variable because c.f is not
                 // writable.
```

Comment:
Consider removing this comment since the logic (and the comment) have moved to `CanModifyReadonlyField()`.

Reply:
> Consider removing this comment since the logic (and the comment) have moved to ```CanModifyReadonlyField()```.

I intentionally left this comment here as well. For the benefit of the reader.